src/common/core/antibot/files/capjs.html

```
@@ -0,0 +1,370 @@
+{-raw-}<!doctype html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)